packages/go-musicfox/build.sh

```diff
@@ -0,0 +1,38 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/go-musicfox/go-musicfox
+TERMUX_PKG_DESCRIPTION="A netease music player in terminal."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@anhoder"
+TERMUX_PKG_VERSION="4.4.0"
+TERMUX_PKG_DEPENDS="libc++, libflac, libvorbis"
+TERMUX_PKG_SRCURL=git+https://github.com/go-musicfox/go-musicfox
+TERMUX_PKG_GIT_BRANCH=master
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_UPDATE_TAG_TYPE="newest-tag"
+TERMUX_PKG_UPDATE_VERSION_REGEXP="\d+\.\d+\.\d+"
+
+termux_step_post_get_source() {
+	git pull
+}
+
+termux_pkg_auto_update() {
+	# Get the newest tag:
+	local tag
+	tag="$(termux_github_api_get_tag "${TERMUX_PKG_SRCURL}" "${TERMUX_PKG_UPDATE_TAG_TYPE}")"
+	# check if this is not a release:
+	if grep -qP "^v${TERMUX_PKG_UPDATE_VERSION_REGEXP}\$" <<<"$tag"; then
+		termux_pkg_upgrade_version "$tag"
+	else
+		echo "WARNING: Skipping auto-update: Not a release($tag)"
+	fi
+}
+
+termux_step_make() {
+	termux_setup_golang
+
+	cd $TERMUX_PKG_SRCDIR/
```

Comment:
Please use `TERMUX_PKG_BUILD_IN_SRC=true` in the control variables above if you wish to build in the source directory.
```suggestion
```